### DISCUSSION TITLE
Should support color values without # when selecting HEX_FORMAT

### DISCUSSION THREAD

**Discussion by tranthanhhoang060391** [SofianChouaib/alwan#16] [Ideas]
Hi,
Thank you for your great library.

When playing with the library, I noticed that when I select the Hex format, and enter the `ff0000` to the input field, it will show black
<img width="268" alt="image" src="https://github.com/sefianecho/alwan/assets/9476360/62566e5c-a109-49d7-a5ae-3c051e2846d1">
When changing it to `#ff0000`, it will show the correct color

When selecting the Hex format, could you consider supporting the color value without a #?

**Comment by sefianecho** [comment]
Yes, I'll try to make the # character optional
